web/src/layouts/MainLayout.vue

```diff
@@ -162,6 +162,11 @@ export default {
         icon: "people",
         link: "/user",
       },
+      {
+        title: t("menu.role"),
+        icon: "manage_accounts",
+        link: "/role",
```

Comment:
This should be pluralized.

Reply:
[copy](https://github.com/zinclabs/zinc/blob/main/web/src/layouts/MainLayout.vue#L161)